samples/gitchurn/src/gitChurnMain/kotlin/GitRepository.kt

```diff
@@ -60,4 +60,19 @@ class GitRepository(val location: String) {
             }
         }
     }
+
+    fun myCommits() = memScoped {
+        val walk = GitRevwalk(handle, sort = GIT_SORT_TOPOLOGICAL or GIT_SORT_TIME)
+        walk.repush()
```

Comment:
The code below is ill-formatted.
Fortunately, Kotlin defines some kind of standard formatting convention: https://kotlinlang.org/docs/reference/coding-conventions.html#formatting

You can use Code -> Reformat Code feature in IntelliJ IDEA to make your code better conform the conventions.